src/pages/projects.js

```diff
@@ -411,11 +412,12 @@ const ProjectDescriptions = () => (
         </div>
         <div className="project-description">
           <p>
-            Major:
+            <b>Major:</b>{" "}
             <a href="https://github.com/libbpf/libbpf">
               <b>libbpf</b>
-            </a>{" "}
-            Emerging:
+            </a>
+            <br />
+            <b>Emerging:</b>{" "}
```

Comment:
Tiny nit, but otherwise good to go. Could we remove the empty `Emerging:` in here for now if we don't have anything to list? Rest LGTM, thanks!

Reply:
Done